### DISCUSSION TITLE
Encryption: how to get it working

### DISCUSSION THREAD

**Discussion by Juelicher-Trainee** [sboesen/remotely-sync#7] [Q&A]
Hi,
I am quite a bit inexperienced but I got the plugin working (check: access). I even managed to upload one single note (2 files, as with an attachment). I could see those files in the browser on storj.io.
But after that sync did not work due to password/phrase issues.
I wanted to have a encrypted sync anyway, so I deleted those files (from the Obsidian vault) and added others.
These do not sync, regardless whether I specify a password in the plugin settings or not. (error: "no local password", "manual, error while checking password").
Regrettably, it is not clear for me what to enter into the Encryption Password field in Plugin settings and why doesn't it work without one.
Possibly, my whole setup is wrong on storj.io.
Any help appreciated. Thank you.

**Comment by sboesen** [comment]
Hello, so sorry to hear about this. Are you still experiencing issues? Not sure why I didn't get a notification for this one.

**Comment by Juelicher-Trainee** [comment]
Thank you Stefan,
miraculously the problem went away and now syncing seems to work and the encryption, too.
Thank you for responding.

**Comment by sboesen** [comment]
Fantastic, that's great to hear & sorry again for the delay! Thank you for following up, closing the issue.
